src/components/ListTemplate/searchAndSortTemplate.js

```diff
@@ -0,0 +1,72 @@
+import { get } from 'lodash';
+
+import {
+  capitalize,
+  formatUserName, getFieldMatched,
```

Comment:
Move `getFieldMatched` to the next line, please

Reply:
Done